confd/templates/karaf/featuresBoot/opennms-es-rest.tpl

```diff
@@ -0,0 +1,7 @@
+#
+# DO NOT CHANGE THIS FILE! GENERATED FROM confd
+#
+{{$base := "/elasticsearch-rest-forwarder" -}}
+{{if and (exists (print $base "/enabled")) (eq (getv (print $base "/enabled") "") ("true")) -}}
```

Comment:
Can use the default here rather than checking for exists first. This pattern can be used in a few more templates in this review that do the same thing, just makes it a bit easier to read.
```suggestion
{{if (eq (getv (print $base "/enabled") "false") "true") -}}
```